bindata/network/multus/multus.yaml

```diff
@@ -84,6 +84,24 @@ data:
       echo "Failed to copy files in ${sourcedir} to $DESTINATION_DIRECTORY"
       exit 1
     fi
+  allowlistcopy.sh: |-
```

Comment:
we better collect this file in must gather

Reply:
Will add a PR to must gather for this one